src/game/enemies/yado.c

```diff
@@ -159,11 +159,11 @@ void Task_8055084(void)
             if (cheese->s.hitboxes[1].index != HITBOX_STATE_INACTIVE) {
                 if (HB_COLLISION(x, y, s->hitboxes[0], I(cheese->posX), I(cheese->posY), cheese->s.hitboxes[1])) {
                     if (IS_MULTI_PLAYER) {
-                        struct UNK_3005510 *unk = sub_8019224();
-                        unk->unk0 = 3;
-                        unk->unk1 = yado2->base.regionX;
-                        unk->unk2 = yado2->base.regionY;
-                        unk->unk3 = yado2->base.id;
+                        struct RoomEvent *room_event = CreateRoomEvent();
```

Comment:
More minor, if we are gonna go with "Room Event" as a type, would prefer the local as `roomEvent` since that's the style we use in this repo